janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementSystem.java

```diff
@@ -148,6 +152,9 @@
     private boolean graphShutdownRequired;
     private boolean isOpen;
 
+    private final static String firstDash = "------------------------------------------------------------------------------------------------\n";
+    private final static String dashBreak = "---------------------------------------------------------------------------------------------------\n";
```

Comment:
As these variables are constants now, please rename them with capital letters (I.e. like `FIRST_DASH` and `DASH_BREAK`)